net-misc/asterisk-base/files/initd-3

```diff
@@ -0,0 +1,313 @@
+#!/sbin/openrc-run
```

Comment:
Honestly, the size of this script also asks for it to be part of an external package.